README.md

```diff
@@ -107,14 +178,24 @@ Imagine that the squares of an ordinary checkerboard are numbered in two differe
 
 **a)**  Given a particular R and C, find the corresponding N.  That is, write an expression for variable N containing variables R and C.
 
+var r = 7;
+
+var c = 7;
+
+r * c = n + 7 * 2
+
 **b)**  Given N, find R.  Write an expression for R which contains N.
 
+n - 7 * 2 = r
+
 **c)**  Given N, find C.  Write an expression for C which contains N.
 
+n - 14 / 2 = c
+
```

Comment:
Again, we wouldn't want to use defined numbers here. For this one, we would use modulo, and Dan's answer is as follows: c = n%8;